### PR TITLE
Chunk groups of indices to avoid overlong queries.

### PR DESCRIPTION
**Pull request by DailyDreaming**
Addresses: https://github.com/HumanCellAtlas/data-store/issues/2204

### CODE REVIEW

File: dss/api/subscriptions_v1.py [hunk 1]
```diff
@@ -173,14 +173,24 @@ def delete(uuid: str, replica: str):
...
 
-def _unregister_percolate(es_client: Elasticsearch, uuid: str):
-    response = es_client.delete_by_query(index="_all",
-                                         doc_type=ESDocType.query.name,
-                                         body={"query": {"ids": {"type": ESDocType.query.name, "values": [uuid]}}},
-                                         conflicts="proceed",
-                                         refresh=True)
-    if response['failures']:
-        logger.error("Failed to unregister percolate query for subscription %s: %s", uuid, response)
+def _unregister_percolate(es_client: Elasticsearch, subscribed_indexes: List[str], uuid: str):
```
Comment (amarjandu) [inline dss/api/subscriptions_v1.py]: is subscribed_indexes used?

File: dss/api/subscriptions_v1.py [hunk 2]
```diff
@@ -173,14 +173,24 @@ def delete(uuid: str, replica: str):
...
+    indices = es_client.indices.get('*').keys()
+
+    def chunked(input_list, chunk_size=20):
```
Comment (amarjandu) [inline dss/api/subscriptions_v1.py]: was chuck size something arbitrary? or from ES docs?

Comment (DailyDreaming) [reply dss/api/subscriptions_v1.py]: It included everything.

**Comment by DailyDreaming** [comment]
Closing this for the moment.  I was under the impression that `_all` was expanded, but if it isn't, then it's already the appropriate fix (staging does not currently have this).

**Pull request closed without merging**